pull_request_template.md

```diff
@@ -74,15 +74,21 @@
 	- Should be named `Contents`, not `Table of Contents`.
 	- Should be the first section in the list.
 	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
+    - Must not feature `Contributing`, `License` or `Footnotes` sections.
 - Has an appropriate license.
 	- **We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.**
 		- Tip: You can quickly add it to your repo by going to this URL: `https://github.com/<user>/<repo>/community/license/new?branch=master&template=cc0-1.0` (replace `<user>` and `<repo>` accordingly).
 	- A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are WTFPL and [Unlicense](https://unlicense.org).
 	- Place a file named `license` or `LICENSE` in the repo root with the license text.
 	- **Do not** add the license name or text to the readme. GitHub already shows the license name at the top of the repo.
+    - A section named `Licence` or `License` pointing to the full license text can still be added at the end of the readme.
+    - That section should not be featured in the Table of Contents.
 	- To verify that you've read all the guidelines, please comment on your pull request with just the word `unicorn`.
 - Has [contribution guidelines](awesome.md#include-contribution-guidelines).
 	- The file should be named `contributing.md`. Casing is up to you.
+    - It should be linked at the bottom of the readme under a section titled `Contribute` or `Contributing`.
```

Comment:
I don't think we should require this. Some might want to have a contribution link at the top or somewhere else.

Reply:
OK. Do you want me to change this to suggest accepted possibilities to the user? What do you think of:

> - It can optionally be linked from the README in a dedicated section titled `Contribute` or `Contributing`, positionned at the top or bottom of the main content.

Reply:
👍🏻 But it could be consistent and only allow one of the wordings.

Reply:
Done in https://github.com/sindresorhus/awesome/pull/1860/commits/1244972781b64c2a46fca8714f4f6448668463d3 . 

I went with `Contributing` for the strict naming of that section, because it echoes the name of the `contributing.md` file.